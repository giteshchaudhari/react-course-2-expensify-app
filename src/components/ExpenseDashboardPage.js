import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesAmountTotal from './ExpensesTotal';


const ExpenseDashboardPage=()=>(
    <div>
        <ExpensesAmountTotal/>
        <ExpenseListFilters/>
        <ExpenseList/>
       
    </div>
);

export default ExpenseDashboardPage;