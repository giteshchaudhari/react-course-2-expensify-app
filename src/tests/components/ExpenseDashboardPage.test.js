import React from 'react'
import ExpenseDashboardPage from './../../component/ExpenseDashboardPage';
import { shallow } from 'enzyme';


test('shouldd render dashboard',()=>{
    const wrapper=shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot();
})