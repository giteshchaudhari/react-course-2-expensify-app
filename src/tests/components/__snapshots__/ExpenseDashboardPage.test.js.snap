// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shouldd render dashboard 1`] = `
<div>
  <Connect(ExpensesAmountTotal) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
